### PR TITLE
feat: Updated video banner

### LINKED ISSUE #260: Re-record landing page video
## Use Case

_Please provide a use case to help us understand your request in context_

## Solution

_Please describe your ideal solution_

## Alternatives

_Please describe any alternatives you've considered, even if you've dismissed them_

### PR DESCRIPTION
**Pull request by ShrimpCryptid**
Closes #260, re-recording the video banner using guidance from @lynwilhelm!

*Estimated review size: tiny, 5 minutes*

## Changes
Re-recorded the new video banner and did some light editing. The new video loops seamlessly, is recorded at higher volume resolution, and is about half the size of the original banner (~6.62 MB compared to 15 MB).

Video of the new banner in action:

https://github.com/allen-cell-animated/website-3d-cell-viewer/assets/30200665/730e5c2a-a00e-43eb-a3c0-af2ffb661809

### CODE REVIEW

**Comment by ShrimpCryptid** [comment]
Taking this back to draft because it looks bad on mobile!

**Comment by toloudis** [comment]
I have an idea to try this on my work PC that might be able to do time series + turntable at the same time with good enough performance.  Can you give me the file url and approximate settings that were used?  Just to see if I can get something good...

**Comment by ShrimpCryptid** [comment]
> I have an idea to try this on my work PC that might be able to do time series + turntable at the same time with good enough performance. Can you give me the file url and approximate settings that were used? Just to see if I can get something good..

URL, I recorded at around frame 73: https://dev-aics-dtp-001.int.allencell.org/website-3d-cell-viewer/viewer?url=https%3A%2F%2Fdev-aics-dtp-001.int.allencell.org%2Fdan-data%2F3500005818_20230811__20x_Timelapse-02%28P27-E7%29.ome.zarr

Settings looked like this, though I also played with the density and brightness some:

image

Raw video:

https://github.com/allen-cell-animated/website-3d-cell-viewer/assets/30200665/c70ace2b-b3c0-4d86-8148-1d535dcb4e21

File: website/components/LandingPage/index.tsx [hunk 1]
```diff
@@ -326,6 +336,7 @@ export default function LandingPage(): ReactElement {
           <video autoPlay={allowMotion} loop muted>
             <source src={BannerVideo} type="video/mp4" />
           </video>
+          <div></div>
```
Comment (rugeli) [inline website/components/LandingPage/index.tsx:339]: Just curious: why is this empty `<div>`?

**Review by rugeli — APPROVED** [review]
I tried to open the url but it didn't work. The screenshot and videos look awesome though! Just one question about the nested empty `div`.

File: website/components/LandingPage/index.tsx (hunk 1 above)
Comment (ShrimpCryptid) [reply website/components/LandingPage/index.tsx:339]: This div is manipulated by lines 58-64 to have the gradient fill!

**Review by interim17 — APPROVED** [review]
Still looks good to me. Last time we checked in about this you were waiting to try some things that @toloudis requested, seems like those are done?

I think it needs main brought in pre-merge! Otherwise, LGTM

**Comment by ShrimpCryptid** [comment]
> Still looks good to me. Last time we checked in about this you were waiting to try some things that @toloudis requested, seems like those are done?
> 
> I think it needs main brought in pre-merge! Otherwise, LGTM

I think the changes were in limbo for now until Dan gets back!

**Pull request merged**